mreg_cli/utilities/api.py

```diff
@@ -0,0 +1,270 @@
+"""Utility functions for mreg_cli.
+
+Due to circular dependencies, this module is not allowed to import anything from mreg_cli.
+And this rule is promptly broken by importing from mreg_cli.outputmanager...
+"""
+
+import json
+import logging
+import os
+import sys
+from typing import TYPE_CHECKING, Any, Dict, List, NoReturn, Optional, cast, overload
+
+import requests
+
+from mreg_cli.config import MregCliConfig
+from mreg_cli.outputmanager import OutputManager
+
+if TYPE_CHECKING:
+    from typing_extensions import Literal
+
+    from mreg_cli.types import ResponseLike
+
+from prompt_toolkit import prompt
+
+from mreg_cli.exceptions import CliError
+from mreg_cli.log import cli_error, cli_warning
+
+session = requests.Session()
+session.headers.update({"User-Agent": "mreg-cli"})
+
+mreg_auth_token_file = os.path.join(str(os.getenv("HOME")), ".mreg-cli_auth_token")
+
+logger = logging.getLogger(__name__)
+
+HTTP_TIMEOUT = 20
+
+config = MregCliConfig()
+
+
+mregurl = config.get("url")
```

Comment:
This value _can't_ be `None`. All the calls to `urljoin` will fail we pass it `None`, as I understand it.

Reply:
Fixed in https://github.com/unioslo/mreg-cli/pull/191/commits/67956443bf3c4ae1ade965238bea5298ee700c26